inst/sql/sql_server/analyses/create_result_concept_table.sql

```diff
@@ -0,0 +1,88 @@
+IF OBJECT_ID('@resultsDatabaseSchema.achilles_result_concept_count', 'U') IS NOT NULL
+  drop table @resultsDatabaseSchema.achilles_result_concept_count;
+
+WITH concepts AS (
+      SELECT
+        CAST(ancestor_concept_id AS VARCHAR)   ancestor_id,
+        CAST(descendant_concept_id AS VARCHAR) descendant_id
+      FROM @vocabDatabaseSchema.concept_ancestor ca
+      UNION
+      SELECT
+        CAST(concept_id AS VARCHAR) ancestor_id,
+        CAST(concept_id AS VARCHAR) descendant_id
+      FROM @vocabDatabaseSchema.concept c
+  ), counts AS (
+  SELECT stratum_1 concept_id, MAX (count_value) agg_count_value
+  FROM @resultsDatabaseSchema.achilles_results
+  WHERE analysis_id IN (2, 4, 5, 201, 225, 301, 325, 401, 425, 501, 505, 525, 601, 625, 701, 725, 801, 825,
+  826, 827, 901, 1001, 1201, 1425, 1801, 1825, 1826, 1827, 2101, 2125, 2301)
+      /* analyses:
```

Comment:
Can you remove these multi-line comments? They can interfere with some SqlRender patterns

Reply:
fixed

Reply:
Sorry -- meant can you remove the comments from the body of the SQL script? We can simply document this in the readme.

Reply:
done